src/Services/Air/AirErrors.js

```diff
@@ -107,8 +107,8 @@ Object.assign(AirRuntimeError, createErrorsList({
   InvalidRequestData: 'Request data is invalid',
   NoValidFare: 'No valid fare for input criteria.',
   TravelersListError: 'Not all BookingTravelers present in list or wrong lookup keys provided',
-  PnrParseError: 'Failed to parse PNR from ticket information request response',
-  GetPnrError: 'Failed to obtain PNR from ticket information',
+  ParseTicketPNRError: ['Failed to parse PNR from ticket information request response', errorCodes.Validation],
+  GetBookingByTicketNumberError: 'Unexpectedly failed to obtain PNR from ticket information',
```

Comment:
Just `Failed to obtain PNR from ticket information` for the same reason.

And I'm not sure about this error. Do we need it? If request to the uAPI failed, why wrap this with another error?